src/tests/simulation/src/main.rs

```diff
@@ -556,7 +561,7 @@ impl Risingwave {
                 tracing::error!("postgres connection error: {e}");
             }
         });
-        if ARGS.kill_compute || ARGS.kill_meta {
+        if ARGS.kill_frontend {
```

Comment:
Why is implicit flush needed when the frontend is possible to crash? (better to add a comment for this)

Reply:
Because `RW_IMPLICIT_FLUSH` is a config per session. When the frontend crashes, we need to reset this config for new session.

Reply:
Oh right, I remembered it. 🥵